aws/cloudformation/ami-manager.js

```diff
@@ -0,0 +1,112 @@
+var response = require('cfn-response');
```

Comment:
done (used the `@file` JSDoc tag we've started using elsewhere in our JS codebase)